### PR TITLE
fix: remove translate relative path to absolute path for runtime path

### LINKED ISSUE #5786: Composer translates relative paths in runtime settings to absolute paths
## Describe the bug

Composer incorrectly changes relative paths to absolute paths. This defeats the purpose of allowing relative paths since if a project folder moves, the path breaks.  

## Version

Current main

## To Reproduce

1. navigate to settings
2. change the runtime path to "../"
3. reload the page. the path will be replaced with an absolute path

## Expected behavior

The relative path should remain unmodified.

## Screenshots
sRmGK578RD

### PR DESCRIPTION
**Pull request by alanlong9278**
## Description
fix: remove translate relative path to absolute path for runtime path.

Have commented the translate function. Call me if there are any question I ignore. @pavolum 

```
 // if (settings?.runtime?.customRuntime && settings.runtime.path && !Path.isAbsolute(settings.runtime.path)) {
    //   const absolutePath = Path.resolve(this.dir, 'settings', settings.runtime.path);

    //   if (fs.existsSync(absolutePath)) {
    //     settings.runtime.path = absolutePath;
    //     await this.updateEnvSettings(settings);
    //   }
    // }
```

## Task Item

closes #5786

### CODE REVIEW

**Comment by pavolum** [comment]
@benbrown I believe we need this translate function logic for the new creation flow to work accordingly when building the adaptive runtime.

**Comment by benbrown** [comment]
I'll check and make sure that it does the translation inline instead of relying on it being done already.

File: Composer/packages/server/src/models/bot/botProject.ts [hunk 1]
```diff
@@ -203,14 +203,14 @@ export class BotProject implements IBotProject {
...
     // Resolve relative path for custom runtime if the path is relative
-    if (settings?.runtime?.customRuntime && settings.runtime.path && !Path.isAbsolute(settings.runtime.path)) {
-      const absolutePath = Path.resolve(this.dir, 'settings', settings.runtime.path);
-
-      if (fs.existsSync(absolutePath)) {
-        settings.runtime.path = absolutePath;
-        await this.updateEnvSettings(settings);
-      }
-    }
+    // if (settings?.runtime?.customRuntime && settings.runtime.path && !Path.isAbsolute(settings.runtime.path)) {
```
Comment (benbrown) [inline Composer/packages/server/src/models/bot/botProject.ts:206]: we should remove these lines of code.

**Comment by benbrown** [comment]
I made a follow-up PR to this one with some additional changes that are necessary.

See: #5950

I believe we should close this PR and use that one, but I will wait for review!

**Comment by alanlong9278** [comment]
Close this pr.

**Pull request closed without merging**